ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/clientlibs/site/css/textinputview.css

```diff
@@ -36,4 +36,6 @@
 
 .cmp-adaptiveform-textinput__questionmark {
 
-}
\ No newline at end of file
+}
+
+[data-cmp-visible=false]
```

Comment:
we can remove this now since it will be captured at form container level.

Reply:
Removed.